spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -17,8 +17,37 @@ include:
 definitions:
 
  - MSG_ACQ_RESULT:
+    id: 0x0014
+    short_desc: Satellite acquisition result
+    desc: |
+      This message describes the results from an attempted GPS signal
+      acquisition search for a satellite PRN over a code phase/carrier
+      frequency range. It contains the parameters of the point in the
+      acquisition search space with the best signal-to-noise (SNR)
+      ratio.
+    fields:
+        - snr:
+            type: float
+            desc: |
+              SNR of best point. Currently dimensonless, but will have
+              units of dB Hz in the revision of this message.
+        - cp:
+            type: float
+            units: chips
+            desc: Code phase of best point
+        - cf:
+            type: float
+            units: hz
+            desc: Carrier frequency of best point
+        - sid:
+            type: u32
+            desc: |
+              Signal identifier of the satellite signal for which acquisition
```

Comment:
How about something like
"Values 0x00000000 through 0x0000001F represent GPS PRNs 1 through 32 respectively (PRN-1 notation).  Other values reserved for future use."